Source/WebCore/platform/graphics/gstreamer/eme/GStreamerEMEUtilities.h

```diff
@@ -107,42 +107,42 @@ class GStreamerEMEUtilities {
         return equalIgnoringASCIICase(keySystem, s_ClearKeyKeySystem);
     }
 
-    static bool isClearKeyUUID(const String& uuid)
+    static bool isClearKeyUUID(CStringView uuid)
     {
-        return equalIgnoringASCIICase(uuid, s_ClearKeyUUID);
+        return equalIgnoringASCIICase(uuid.toString(), s_ClearKeyUUID);
     }
 
     static bool isWidevineKeySystem(const String& keySystem)
     {
         return equalIgnoringASCIICase(keySystem, s_WidevineKeySystem);
     }
 
-    static bool isWidevineUUID(const String& uuid)
+    static bool isWidevineUUID(CStringView uuid)
     {
-        return equalIgnoringASCIICase(uuid, s_WidevineUUID);
+        return equalIgnoringASCIICase(uuid.toString(), s_WidevineUUID);
```

Comment:
Wouldn't it make more sense to have overloads of `equalIgnoringASCICase()` that can take `CStringView` parameters instead of creating temporary `String` copies of the contents?